plugins/joblib_launcher/hydra_plugins/joblib_launcher/conf/hydra/launcher/joblib.yaml

```diff
@@ -0,0 +1,16 @@
+hydra:
+  launcher:
+    class: hydra_plugins.joblib_launcher.JoblibLauncher
+    params:
+      joblib:
+        n_jobs: -1
+        backend: None
```

Comment:
Can you add comments here documenting each of the options? (minimal doc, saying what it is and potentially what are the valid input values).

Reply:
Added in latest round of commits